packages/internals/src/cli/getSchema.ts

```diff
@@ -109,8 +109,7 @@ export async function getSchemaWithPathOptional({
 }
 
 export function printSchemaLoadedMessage(schemaPath: string) {
-  // TODO: this causes https://github.com/prisma/prisma/issues/27005
-  process.stdout.write(dim(`Prisma schema loaded from ${path.relative(process.cwd(), schemaPath)}`) + '\n')
+  process.stderr.write(dim(`Prisma schema loaded from ${path.relative(process.cwd(), schemaPath)}.`) + '\n')
```

Comment:
why is this written to `stderr`?

Reply:
This looks like a change from [another Alberto's pull request](https://github.com/prisma/prisma/pull/28911), it looks like this one unintentionally includes commits from the other one.

Reply:
Good catch. Sorry about that. I removed those merge commits from my other PR.